src/common/TrackedOp.h

```diff
@@ -179,11 +179,14 @@ class TrackedOp {
   const utime_t& get_initiated() const {
     return initiated_at;
   }
-  // This function maybe needs some work; assumes last event is completion time
+
   double get_duration() const {
-    return events.empty() ?
-      0.0 :
-      (events.rbegin()->first - get_initiated());
+    if (events.empty())
+      return 0.0;
```

Comment:
In  TrackedOp(OpTracker *_tracker, const utime_t& initiated) :
 if (tracker->tracking_enabled)
      events.push_back(make_pair(initiated_at, "initiated"));
In fact, there is no change meet events.empty() is true. 
or am i missing something?